vignettes/articles/03-run-stress-test.Rmd

```diff
@@ -65,6 +65,12 @@ This will calculate the stress test for loans and write the results along with a
 log file into the designated output directory, using the default values for all
 detail arguments.
 
+**NOTE:** The default mode will **not return** any objects within the R session.
+We find this is the easiest and most user friendly way to obtain results, but we
+acknowledge that more advanced R users may find this to depart from the
+conventions of other packages. Below we highlight, how to return the results as
```

Comment:
I would not say convention of other packages, this phrase does not make sense and I could banter on on why, instead I would suggest to say sth like>
We alternatively provide the option that the function returns a list holding the result objects. This allows the user to inspect and further analyse the data directly in R.

Reply:
Thinking about it, I believe we should consider a follow up (non blocking for the next week) to return the results as 5 separate objects. If we aim at user friendliness and expect that some users will not have used R much, I think a list type object still presents somewhat of a barrier.

Reply:
a list is the only way to cleanly return >1 objects 🤡  , unless we adjust the function in a way that it only returns 1 result type per run. 
I would say being able to deal with a list is a nicer selector for if one should use the option

Reply:
fine by me, let's see what the users think